src/bobtail/bobtailblock.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_BOBTAIL_BOBTAILBLOCK_H
+#define BITCOIN_BOBTAIL_BOBTAILBLOCK_H
+
+#include "block.h"
+#include "subblock.h"
+
+class CBobtailBlock : public CBlock
+{
+public:
+    std::vector<CSubBlockRef> vdag;
```

Comment:
I think that this would need to be a `CDagForrest` in order to allow the receiver to validate.

Reply:
woudlnt we want the reassembly of the dag from a set of subblocks to be part of the validation?

Reply:
How would you know how to reconstruct the DAG without the ancestor hashes?

Reply:
they are included in the subblocks